CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.3.12b - 2023-05-12
```

Comment:
Adding featurs would not be a B release. 

Releases are done by the team, add as Unreleased please, also follow the format below, add the PR number and @ tag of the author